gds_write/build_defs.bzl

```diff
@@ -81,6 +82,7 @@ def _gds_write_impl(ctx):
                   gds_allow_empty_args +
                   " --out {}".format(final_gds.path),
         tools = depset([ctx.executable._gds_write]),
+        toolchain = "//python:current_py_toolchain",
```

Comment:
shouldn't we be using:
```
toolchains=["@rules_python//python:current_py_toolchain"]
```
as recommended in https://rules-python.readthedocs.io/en/latest/getting-started.html#toolchain-usage-in-other-rules ?